### ISSUE TITLE
Release a new version of khal

### ISSUE THREAD

**Issue by sgibb** [geier/khal#203]
I just reinstalled `khal` on a new machine and it doesn't start anymore. The only difference I could figure out is a new version of `vdirsyncer` (`5.0.1` vs. `4.4.0` on my old machine).

I installed `khal` as follows:
```
cd ~/.virtualenvs
virtualenv khal
./khal/bin/pip install khal
```

Starting `khal` fails:
```
sebastian@debian ~ $ khal -v
Traceback (most recent call last):
  File "/home/sebastian/.virtualenvs/bin/khal", line 9, in <module>
    load_entry_point('khal==0.4.0', 'console_scripts', 'khal')()
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 546, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2666, in load_entry_point
    return ep.load()
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2339, in load
    return self.resolve()
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2345, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/khal/cli.py", line 33, in <module>
    from khal import controllers
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/khal/controllers.py", line 33, in <module>
    from khal.khalendar.exceptions import ReadOnlyCalendarError
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/khal/khalendar/__init__.py", line 23, in <module>
    from .khalendar import Calendar, CalendarCollection  # flake8: noqa
  File "/home/sebastian/.virtualenvs/khal/local/lib/python2.7/site-packages/khal/khalendar/khalendar.py", line 35, in <module>
    from vdirsyncer.storage import FilesystemStorage
ImportError: cannot import name FilesystemStorage
```

**Comment by sgibb** [comment]
Sorry, it was already fixed in https://github.com/geier/khal/commit/28ed8d9396dc5104d2154a89b55690f86732f42c

**Comment by untitaker** [comment]
Argh. I should've coordinated the release of vdirsyncer better with khal's release process, sorry about that.

**Comment by geier** [comment]
my fault I guess, I asked for the breaking change in vdirsyncer...

If it isn't fixed with the 0.5 release, please reopen.
